express/blocks/ratings/ratings.js

```diff
@@ -0,0 +1,517 @@
+}
+
+function submitRating(rating, comment) {
+  const content = {
+    data: [
+      {
+        name: 'Segments',
+        value: Context.get('audiences') ?? '',
+      },
+      {
+        name: 'Locale',
+        value: getLocale(window.location),
+      },
+      {
+        name: 'Rating',
+        value: rating,
+      },
+      {
+        name: 'Timestamp',
+        value: new Date().toLocaleString(),
+      },
+      {
+        name: 'Comment',
+        value: comment,
+      },
+    ],
+  };
+
+  fetch(`https://www.adobe.com/reviews-api/ccx${sheet}`, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(content),
+  });
+
+  let ccxActionRatings = localStorage.getItem('ccxActionRatings') ?? '';
+
+  if (!ccxActionRatings.includes(sheet)) {
+    ccxActionRatings = ccxActionRatings.length > 0 ? sheet : `,${sheet}`;
+  }
+
+  localStorage.setItem('ccxActionRatings', ccxActionRatings);
+}
+
+// Updates the front-end style of the slider.
+function updateSliderStyle($block, value) {
+  const $input = $block.querySelector('input[type=range]');
+  const $tooltip = $block.querySelector('.tooltip');
+  const $sliderFill = $block.querySelector('.slider-fill');
+  const thumbWidth = 60;
+  const pos = (value - $input.getAttribute('min')) / ($input.getAttribute('max') - $input.getAttribute('min'));
+  const thumbCorrect = (thumbWidth * (pos - 0.25) * -1) - 0.1;
+  const titlePos = (pos * $input.offsetWidth) - (thumbWidth / 4) + thumbCorrect;
+  $tooltip.style.left = `${titlePos}px`;
+  $sliderFill.style.width = `${titlePos + (thumbWidth / 2)}px`;
+}
+
+// Implements the slider logic.
+function sliderFunctionality($block) {
+  const $input = $block.querySelector('input[type=range]');
+  const $sliderFill = $block.querySelector('.slider-fill');
+  const $tooltip = $block.querySelector('.tooltip');
+  const $tooltipText = $block.querySelector('.tooltip--text');
+  const $tooltipImg = $block.querySelector('.tooltip--image');
+  const $textarea = $block.querySelector('.slider-comment textarea');
+  const $textareaLabel = $block.querySelector('.slider-comment label');
+  const $stars = Array.from($block.querySelectorAll('.stars'));
+  const $submit = $block.querySelector('input[type=submit]');
+  const $scrollAnchor = $block.querySelector('.ratings-scroll-anchor');
+  const $commentBox = $block.querySelector('.slider-comment');
+  const $timer = createTag('div', { class: 'timer' });
+  // Countdown timer to auto-submit
+  const countdown = (bool) => {
+    if (bool) {
+      $timer.innerHTML = getLottie('countdown', '/express/blocks/ratings/countdown.json', false, true, false, false);
+      let counter = 10;
+      window.ratingSubmitCountdown = setInterval(() => {
+        if (counter > 0) {
+          counter -= 1;
+          // eslint-disable-next-line no-console
+          console.log(`rating will be submitted in: ${counter + 1} seconds.`);
```

Comment:
```suggestion
          // eslint-disable-next-line no-console
          console.log(`rating will be submitted in: ${counter + 1} seconds.`);
```
I think we should remove (or comment) this in the production version.